### PR TITLE
15.0 [2431][ADD] dotfiles

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[2431](https://www.quartile.co/web#id=2431&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: .copier-answers.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+github_enable_makepot: false
+github_enable_stale_action: true
+github_enforce_dev_status_compatibility: false
+include_wkhtmltopdf: false
+odoo_version: 15.0
+org_name: Quartile Limited
+org_slug: qrtl
+rebel_module_groups: []
+repo_description: null
+repo_name: Custom Modules
```
Comment (yostashiro) [inline .copier-answers.yml]:
```suggestion
repo_name: FSB Custom
```

**Review by yostashiro — CHANGES_REQUESTED** [review]
@AungKoKoLin1997 Can you please update your guideline document as necessary.

**Comment by AungKoKoLin1997** [comment]
> @AungKoKoLin1997 Can you please update your guideline document as necessary.

Already updated in guideline document.

**Review by yostashiro — APPROVED** [review]
Thanks!

**Pull request merged**